### PR TITLE
DVCSMP-1743 INTL-532 Add zll dth equivalent

### PR DESCRIPTION
**Pull request by workingmonk**
cc @tpmanley @tylerlange

### CODE REVIEW

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/**
+ *  Copyright 2015 SmartThings
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy]: The copyright date should be 2016

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+def setGenericName(value){
+    if (value != null) {
+        def genericName = "White"
+        if (value < 3300) {
+            genericName = "Soft White"
+        } else if (value < 4150) {
+            genericName = "Moonlight"
+        } else if (value <= 5000) {
+            genericName = "Cool White"
+        } else if (value >= 5000) {
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy]: Could simply to just 'else'

Comment (tpmanley) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy]: I mean simplify

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+def setColorTemperature(value) {
+    setGenericName(value)
+    zigbee.setColorTemperature(value) + ["delay 1500"] + zigbee.colorTemperatureRefresh()
+}
+
+//Naming based on the wiki article here: http://en.wikipedia.org/wiki/Color_temperature
+def setGenericName(value){
+    if (value != null) {
+        def genericName = "White"
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy]: genericName will always be modified below so no point in initializing it

**Comment by tpmanley** [comment]
Just a few minor comments, otherwise looks good. 

:+1:

**Comment by workingmonk** [comment]
@tpmanley fixed based on your recommendation

**Comment by workingmonk** [comment]
@tylerlange any objections on merging in?

**Comment by table-xx** [comment]
+1

**Comment by tylerlange** [comment]
👍

**Pull request merged**